### PR TITLE
fixed phpdoc & typos

### PR DESCRIPTION
**Pull request by mhor**

### CODE REVIEW

File: src/Connection.php [hunk 1]
```diff
@@ -60,6 +61,8 @@ public function __construct($hostname, $port, $connectTimeout = null, $connectPe
      * Sets a manually created socket, used for unit testing.
      *
      * @param Socket $socket
+     *
+     * @return $this
```
Comment (pda) [inline src/Connection.php]: If `return $this` is the correct way to express a method returning its receiver, you could replace `@chainable` here and throughout `src/PheanstalkInterface.php`. As far as I can remember I made up `@chainable` and it's not used by any tools…?

Comment (GrahamCampbell) [inline src/Connection.php]: Yeh, `@return $this` is good. `@chainable` isn't a think.

**Comment by pda** [comment]
Nice, thanks.
Licence vs License is because Pheanstalk was written in Australia. But… I'm living in USA currently so I'll accept the change to “license” ;)
For the copy-pasted method name typo… I have no excuse.

**Comment by GrahamCampbell** [comment]
> “license” ;)

`@license` is the only valid phpdoc. Anything else is wrong. :)

**Comment by pda** [comment]
Well, if somebody from UK can accept that, so can I.

**Comment by pda** [comment]
Merged along with 279ee9b853150c6b8ff9dbfae49debf4615c688b.

**Pull request merged**